__tests__/app.test.js

```diff
@@ -26,3 +26,39 @@ describe("GET /api/topics", () => {
       });
   });
 });
+describe("GET /api/articles/:article_id", () => {
+  test("200: responds with the article matching the input article id passed as a parameter", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body: { article } }) => {
+        expect(article).toEqual(
+          expect.objectContaining({
```

Comment:
Please tweak this a little so you check the rest of the keys match the exact values you're getting back from the db match the obj you declare in your tests - this type of "checking for shape" test is more suited to when we're checking a collection of objects. 
Here because its a GET `/api/articles/1` we can check that all the values are those from article 1